config/initializers/saml.rb

```diff
@@ -1,3 +1,4 @@
 # frozen_string_literal: true
 Settings.saml.certificate = File.read(File.expand_path(Settings.saml.cert_path))
 Settings.saml.key = File.read(File.expand_path(Settings.saml.key_path))
+Settings.saml.certificate_new = File.read(File.expand_path(Settings.saml.cert_new_path))
```

Comment:
Can we make this tolerant of a nil value in settings, so we don't have to specify a second cert in the steady state where we only have one.